src/arrayMethodSort.js

```diff
@@ -3,10 +3,40 @@
 /**
  * Implement method Sort
  */
+const compareAsString = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  } else {
+    return -1;
+  }
+};
+
```

Comment:
Why do you need as many as three different output options, if you only care about more than 0 or not? 🤓

This will be enough 😉👇
 ```
 if (stringA > stringB) {
    return 1;
```

![image](https://user-images.githubusercontent.com/108505176/185185320-81e3ece6-3384-4df6-b7de-5fa44e63c142.png)